mobile/src/Teacher/screens/Explore/DetailScreen.js

```diff
@@ -0,0 +1,118 @@
+import React, { Fragment, useState } from 'react';
+import NavBarView from './NavBarView'
+import { fontFamilies, fonts } from '../../../utils/theme'
+import { FlatList } from 'react-native-gesture-handler'
+import {
+    StyleSheet,
+    Text,
+    SafeAreaView,
+    View,
+    Button,
+    TouchableOpacity,
+    Pressable
+} from 'react-native';
+import { inOut } from 'react-native/Libraries/Animated/src/Easing';
+import Card from './QuestionCard';
+import { createStackNavigator } from '@react-navigation/stack';
+import ButtonBack from '../../../components/ButtonBack/index';
+import Aicon from 'react-native-vector-icons/FontAwesome';
+import LinearGradient from 'react-native-linear-gradient'
+
+// import GameDetailsScreen from './GameDetails';
+// import Games from '../Games';
+
+const detailList = ({route, navigation}) => {
+
+    const { game } = route.params;
+
+    const questions = [game.q1,game.q2,game.q3,game.q4,game.q5].filter( (question) => !!question ) 
+    
+    const parsedQuestions =  questions.map( (question) => JSON.parse(question)  )
+
+    return(
+            <SafeAreaView style={{ flex: 1, width: '100%'}}>
```

Comment:
I've been trying to avoid inline styling unless it's the case where the component requires it. This helps with code readability. Would be nice if these styles to be moved to `styles` objects and then use it here. Something like:
```javascript
 <SafeAreaView style={styles.mainContainer}>
...
const styles = StyleSheet.create({
         flex: 1,
         width: '100%'
})
```

Also, notice it's reusing `mainContainer` since that's not used in this file.